docs/configuration/features/binary_features.md

```diff
@@ -59,46 +36,41 @@ encoder:
+
+{% set encoder_passthrough = get_encoder_schema("binary", "passthrough") %}
+{{ render_yaml(encoder_passthrough, parent="encoder") }}
+
 There are no additional parameters for the `passthrough` encoder.
 
 ### Dense Encoder
 
-For the `dense` encoder these are the available parameters.
-
-- `num_layers` (default `1`): this is the number of stacked fully connected layers that the input to the feature passes
-through. Their output is projected in the feature's output space.
-- `output_size` (default `256`): if `output_size` is not already specified in `fc_layers` this is the default
-`output_size` that will be used for each layer. It indicates the size of the output of a fully connected layer.
-- `use_bias` (default `true`): boolean, whether the layer uses a bias vector.
-- `weights_initializer` (default `'glorot_uniform'`): initializer for the weights matrix. Options are: `constant`,
-`identity`, `zeros`, `ones`, `orthogonal`, `normal`, `uniform`, `truncated_normal`, `variance_scaling`, `glorot_normal`,
-`glorot_uniform`, `xavier_normal`, `xavier_uniform`, `he_normal`, `he_uniform`, `lecun_normal`, `lecun_uniform`.
-- `bias_initializer` (default `zeros`):  initializer for the bias vector. Options are: `constant`, `identity`, `zeros`,
-`ones`, `orthogonal`, `normal`, `uniform`, `truncated_normal`, `variance_scaling`, `glorot_normal`, `glorot_uniform`,
-`xavier_normal`, `xavier_uniform`, `he_normal`, `he_uniform`, `lecun_normal`, `lecun_uniform`.
-- `norm` (default `null`): normalization applied at the beginnging of the fully-connected stack. If a `norm` is not already specified for the `fc_layers` this is the default `norm` that will be used for each layer. One of: `null`, `batch`, `layer`, `ghost`. See [Normalization](../combiner.md#normalization) for details.
-- `norm_params` (default `null`): parameters passed to the `norm` module. See [Normalization](../combiner.md#normalization) for details.
-- `activation` (default `relu`): if an `activation` is not already specified in `fc_layers` this is the default
-`activation` that will be used for each layer. It indicates the activation function applied to the output.
-- `dropout` (default `0`): dropout rate
+The `dense` encoder passes the raw binary values through a fully connected layer. Inputs of size `b` are transformed to size `b x h`.
+
+{% set encoder_dense = get_encoder_schema("binary", "dense") %}
+{{ render_yaml(encoder_dense, parent="encoder") }}
+
+Parameters:
+
+{{ render_fields(schema_class_to_fields(encoder_dense, exclude=["type"]), details=details) }}
 
 ## Binary Output Features and Decoders
 
+Binary output features can be used when a binary classification needs to be performed or when the output is a single probability. There are two decoders available: `regressor` and `passthrough`.
+
+Example binary output feature using default parameters:
+
 ```yaml
 name: binary_column_name
 type: binary
 reduce_input: sum
 dependencies: []
+calibration: false
```

Comment:
Nice catch. I jut did a quick check of all the params for binary output feature as follows:

```
from ludwig.schema.features.binary_feature import BinaryOutputFeatureConfig
print(BinaryOutputFeatureConfig())
```

Looks like the one other param worth calling out is `threshold`, which is no longer part of the loss, as it was before. I would add its default `threshold: 0.5` in the YAML here and also add its param description to the list just below.

Reply:
Done!